feeds/src/components/oracleTable/OracleTable.component.tsx

```diff
@@ -36,56 +33,46 @@ const OracleTable: React.FC<Props> = ({
   }, [ethGasPrice])
 
   useEffect(() => {
-    const mergedData = networkGraphNodes
-      .filter((node: any) => node.type === 'oracle')
-      .map((oracle: any) => {
-        const state = _.find(networkGraphState, { sender: oracle.address })
-        return {
-          oracle,
-          state,
-          key: oracle.id,
-        }
-      })
-    setData(mergedData)
-  }, [networkGraphState, networkGraphNodes])
+    setData(latestOraclesState)
+  }, [latestOraclesState])
 
   const columns = [
     {
       title: 'Oracle',
-      dataIndex: 'oracle.name',
+      dataIndex: 'name',
       key: 'name',
-      sorter: (a: any, b: any): number =>
-        a.oracle.name.localeCompare(b && b.oracle && b.oracle.name),
+      sorter: (a: any, b: any): number => a.name.localeCompare(b && b.name),
```

Comment:
How do you feel about extracting an interface for the columns so we can reduce the type annotations? e.g.

```typescript
interface Column {
   title: string
   dataIndex: string
   key: string
   sorter: (a: any, b: any) => number
   defaultSortOrder?: string
   render?: (timestamp: number) => string
}

const columns: Column[] = [
   {
      title: 'Oracle',
      dataIndex: 'name',
      key: 'name',
      sorter: (a, b) => a.name.localeCompare(b && b.name),
   }
   // ...
]
```

Reply:
Imported interface straight from antd library

Reply:
👍 sweet. Event better 😄